src/custom.scss

```diff
@@ -0,0 +1,12 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+// change the theme
+$theme-colors: (
+  'ner-red': #ef4345
```

Comment:
if you're gonna add one, please add the others from [the Brand Guideline doc](https://docs.google.com/document/d/1bV-TGorNeGbn7BhrwP9_nhSb_qmfs5wxJeJY5AfekDo/edit)